addon/utils/clamp.js

```diff
@@ -242,5 +239,14 @@ export default (function(win, doc) {
 
     // call the callback with whether or not the text was truncated
     cb(lineCount > lineClamp);
+
+    function cleanup() {
+      thisNode = null;
+      textNode = null;
+      measure = null;
+      line = null;
+    }
```

Comment:
Can you please move the implementation of `cleanup` nearer the initialization section?

nitpick: This file follows a style that places the name of the function in a comment following its closing curly. Can you add `// function cleanup` to the last line please?